src/components/SearchBar.js

```diff
@@ -54,6 +58,10 @@ type SearchBarProps = {
   buttonLabel: string;
   /** If set, this is the route to navigate to upon executing a search. By default, no navigation will occur when searching. */
   route: string | null;
+  /**
+   * If set, a new signal of this type would be added when an autocomplete item is selected.
+   */
+  autoCompleteSignalType: string | null;
```

Comment:
Indicate `autoCompleteSignalType` is optional with the flow annotation.
```suggestion
  autoCompleteSignalType?: string | null;
```